mongodb-replica-set-centos/README.md

```diff
@@ -0,0 +1,82 @@
+# Install MongoDB Replica Set
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fmongodb-replica-set-centos%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="
+http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fmongodb-replica-set-ubuntu%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+
+This template deploys a MongoDB Replica Set on CentOS and enables Zabbix monitoring, and allows user to define the number of secondary nodes. The replica set has a primary node, 2 secondary nodes by default.
+
+This template also allows you to input your existing zabbix server IP address to monitor these MongoDB nodes.
+
+The replica set nodes are exposed on public IP addresses that you can access through SSH on the standard port, also mongodb port 27017 open.
+
+The nodes are under the same subnet 10.0.1.0/24. The primary node ip is 10.0.1.240, the secondary nodes ip address start from 10.0.1.4. For example:
+
+primary node ip: 10.0.1.240
+
+secondary node 1 ip: 10.0.1.4
+
+secondary node 2 ip: 10.0.1.5
+
+
+##Important Notice
+Each VM of the replica set uses raid0 to improve performance. We use 4 data disks on each VM for raid0. The size of data disks(setup raid0) on each VM are determined by yourself. However, there is size of data disks limit per the VM size. Before you set the size of data disks, please refer to the link https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-linux-sizes/ for the correct choice.
+
+
+
+##After deployment, you can do below to verify if the replica set really works or not:
+
+1 SSH connect to primary node, execute below
```

Comment:
This doesn't render as a list in markdown. It'd be great if you could take a look at the rendered file and fix the lists for better readability 

https://github.com/251744647/azure-quickstart-templates/blob/4cedc900454ed6be9c9c10728a49b5f8b82fffd8/mongodb-replica-set-centos/README.md

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet